### DISCUSSION TITLE
Laravel 10

### DISCUSSION THREAD

**Discussion by elijahcruz12** [dedoc/scramble#110] [General]
Hello guys! I'm trying to upgrade to Laravel 10, however I noticed the contracts requirement in your `composer.json` hasn't been updated to allowing for Laravel 10. I am asking if you might be doing so soon, because I really would love to upgrade the Laravel version on the api of the game I'm creating to 10 so I can use Pennant. Thank you!

**Comment by romalytvynenko** [comment]
@elijahcruz12 Added Laravel 10 support in 0.7.2!
